docs/api.md

```diff
@@ -70,6 +70,50 @@ Example:
 };
 ```
 
+__V2 Order fields__
+
+|Field Name|JSON type|Description|
+|----------|---------|-----------|
+|isBuy|boolean|If the order is a buy order|
+|isDecreaseOnly|boolean|(Optional)If the Stop-Limit order is tied to an existing Isolated Position|
+|baseMarket|string|The Solo base [market](https://docs.dydx.exchange/#/overview?id=markets)|
+|quoteMarket|string|The Solo quote [market](https://docs.dydx.exchange/#/overview?id=markets)|
+|amount|string|The amount of token being offered in base units|
+|limitPrice|string| The worst base/quote price at which the transaction will be accepted|
+|triggerPrice|string|(Optional)The price at which the order will go to market.|
+|limitFee|string| Makers will pay 0% fees. Takers with greater than or equal to .5Eth in the transaction will pay .15% of ETH-DAI and ETH-USDC transactions and .05% for DAI-USDC transactions.
+For transactions below .5Eth they will pay .50% fees.
+|makerAccountNumber|string|The Solo [account number](https://docs.dydx.exchange/#/overview?id=markets) of the Maker|
+|makerAccountOwner|string|The Ethereum address of the Maker.|
+|expiration|string|The time in unix seconds at which this order will be expired and can no longer be filled. Use `"0"` to specify that there is no expiration on the order.|
+|salt|string|A random number to make the orderHash unique.|
+|typedSignature|string|The signature of the order.|
+
+Example:
+```json
+{
+    "isBuy": "true",
+    "isDecreaseOnly": "false",
+    "baseMarket": "WETH",
+    "quoteMarket": "DAI",
+    "amount": "10000000000",
+    "limitPrice": "20.3",
+    "triggerPrice": "0",
+    "limitFee": ".15",
+    "makerAccountNumber": "111",
+    "makerAccountOwner": "0x3E5e9111Ae8eB78Fe1CC3bb8915d5D461F3Ef9A9",
+    "expiration": "4294967295",
+    "salt": "100",
+    "typedSignature": "0xd9c006cf9066e89c2e75de72604751f63985f173ca3c69b195f1f5f445289a1f2229c0475949858522c821190c5f1ec387f31712bd21f6ac31e4510d5711c2681f00"
+  },
+};
+
+Note: The tick size is 0.01 for ETH-DAI transactions and 0.01e-12 for ETH-USDC and DAI-USDC transactions.
```

Comment:
0.0001e-12 for DAI-USDC

Reply:
Also explain the e-12 for this and the price

Reply:
Not sure what the best explanation would be - the decimal relationship of USDC vs. ETH and DAI?